src/debug/daccess/gcinterface.dac.h

```diff
@@ -0,0 +1,5 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#include "../gc/gcinterface.dac.h"
```

Comment:
You're right, that is surprising - I guess the GC directory is still on the include path and `#include "gcinterface.dac.h"` includes the original header and not this one? I'll fix this header.